services/openvpn/service/factory.go

```diff
@@ -135,6 +136,10 @@ func newSessionConfigNegotiatorFactory(networkOptions node.OptionsNetwork, servi
 		return &OpenvpnConfigNegotiator{
 			natEventGetter: natEventGetter,
 			vpnConfig: &openvpn_service.VPNConfig{
+				DNS: providerIP(net.IPNet{
```

Comment:
Should be just DNS endpoint `DNS: "10.0.0.1:53"` or `DNS: ["10.0.0.1:53", "10.0.0.2:53"]`
What's the point to communicate virtual network information?

Reply:
It's just an IP `10.8.0.1`, nothing more.

Reply:
Oh, You're calculating it here. Lets just inject it instead of duplicated calculation

Reply:
Changed